src/pages/MemoriesPage.spec.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react'
+import MemoriesPage from './MemoriesPage'
+
+describe('MemoriesPage', () => {
+  it('renders two listItems', () => {
+    render(<MemoriesPage />)
+
+    const memoryItems = screen.getAllByRole('listitem')
+    expect(memoryItems).toHaveLength(2)
```

Comment:
I don't think this test passes.
I think you forgot the `memories` property.

Reply:
Update test with memories